qpup_vision/src/person_tracker.cpp

```diff
@@ -0,0 +1,103 @@
+#include <Eigen/Geometry>
+#include <actionlib/client/simple_action_client.h>
+#include <ar_track_alvar_msgs/AlvarMarkers.h>
+#include <geometry_msgs/Pose.h>
+#include <move_base_msgs/MoveBaseAction.h>
+#include <ros/ros.h>
+#include <std_msgs/String.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <tf2_ros/buffer.h>
+#include <tf2_ros/transform_listener.h>
+
+#include <cmath>
+#include <memory>
+
+using MoveBaseClient = actionlib::SimpleActionClient<move_base_msgs::MoveBaseAction>;
+
+ros::Subscriber ar_track_sub;
+std_msgs::String state;
+ros::Publisher state_pub;
+move_base_msgs::MoveBaseGoal goal;
+std::unique_ptr<MoveBaseClient> ac;
+
+tf2_ros::Buffer buffer_;
+
+enum class State { FOLLOW, STOP, UNKNOWN };
```

Comment:
```suggestion
enum class State { UNKNOWN , FOLLOW, STOP };
```

not that it really matters, but default constructing will zero-init (first index)